book/website/community-handbook/community-calls/community-calls-onboarding.md

```diff
@@ -3,7 +3,7 @@
 
 ![Alt: Two people, one woman with a long orange shirt and one guy with a blue shirt and a tie, and a large Welcome! sign at the back](https://i.imgur.com/NXOPPc4.jpg)
 
-#### **Welcome to The Turing Way!**
+## **Welcome to The Turing Way!**
```

Comment:
I would actually suggest removing this heading altogether, it looks weird in the middle of the book.

https://deploy-preview-4144--the-turing-way.netlify.app/community-handbook/onboarding-offboarding/onboarding-general-calls